app/views/userProfile.scala.html

```diff
@@ -9,7 +11,39 @@
 @main(title) {
     @navbar(user, user.map(u=> "/contribution/" + u.username))
     <div class="container">
-        <div class="row" id="maprow">
+
+        @if(user && user.get.role.getOrElse("") != "Anonymous") {
+            <div class="row" id="profile-container">
+                <div id="username">@user.get.username</div>
+                <div id="org-container">
+                    @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
+                        <div id="org-name">@OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head)</div>
+                    } else {
+                        <div id="org-name">@Messages("dashboard.no.org")</div>
+                    }
+                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right" data-content="@Messages("dashboard.edit.info")">
```

Comment:
Clicking the link to email could be logged?

Reply:
@misaugstad Hmmm I'm having trouble doing this. I am trying to do this logging starting on line 224 (of my latest commit) of userProfile.scala.html, however, both the clicks on the email link and clicks on the edit org button are triggering this onclick()... any suggestions?

Reply:
It looks like you're using the HTML ID of the drop-down menu, and then applying it to all links contained within that div. I think that if you just add an ID attribute to the email link, you could probably just reference that ID specifically!

Reply:
Hey @misaugstad, unfortunately, I've tried that already and it's not working... although I could be misinterpreting what you are saying. I just pushed up some changes, is this what you meant?

Reply:
yeah that's what I meant. weird that that isn't working! I'll take a look hopefully later this week and investigate!

Reply:
@misaugstad Right! So weird! Any luck on your end?

Reply:
Hey @misaugstad! Just wanted to follow up on this.

Reply:
hehe yep, I actually just reread this comment a couple minutes ago! :) I'm hoping to test this today or tomorrow!

Reply:
Yep, the issue is that the content in the popover is created on mouseover and destroyed on mouseout (despite being referred to as 'show' and 'hide' in the code). We now create the listener when we make the popover, so the listener will be created every time the popover is created. It's working now :)